static/js/util/currency.js

```diff
@@ -1,17 +1,50 @@
 // @flow
 import cc from 'currency-codes';
 import R from 'ramda';
+import iso3166 from 'iso-3166-2';
+
+export const excludedCurrencyCodes = [
```

Comment:
my inclination would be to file an issue for that and go with the hardcoded list for now